packages/enterprise-search/docs/release-notes/html_docs/raw/release-notes.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
Don't want to commit this file. Should be good to merge after removing.

Reply:
Good catch thanks